CSS files/components/buttons/buttons.css

```diff
@@ -1,16 +1,18 @@
 
 .btn{
-    padding:0.8rem 1.2rem;
+    padding:8px 1rem;
     border-radius: .8rem;
     border:none;
     cursor: pointer;
+    font-size: 1.2rem;
 }
 
 .btn-large{
-    padding:8px 1.4rem;
-    border-radius: 15px;
+    padding:10px 1.2rem;
+    border-radius: 8px;
     border:none;
     cursor: pointer;
+    font-size: 1.6rem;
 }
 
 .btn-primary{
```

Comment:
Do not keep empty classes with no styles. Suggestion: Add generic styles to .btn and add extra styling to btn-primary, btn-secondary etc.

Reply:
Got it Arpan, I will refactor this.